.changeset/strange-apes-leave.md

```diff
@@ -0,0 +1,11 @@
+---
+"@jpmorganchase/uitk-core": patch
+---
+
+Apply
+aria-disabled
+to
+Pill
+when
+it's
+disabled
```

Comment:
is this format intended?

Reply:
It's what prettier chose. I think there's an issue with our prettier config and changesets

Reply:
Looking at this on main, if I manually edit and run prettier it's fine with being a long line but not sure why it decides to wrap sometimes.